### PR TITLE
knapsack accept slices

### PR DESCRIPTION
**Pull request by victor-prokhorov**
make the function accept both vectors and slices

### CODE REVIEW

**Comment by remlse** [comment]
LGTM :+1:

**Comment by remlse** [comment]
@victor-prokhorov I think the [example file](https://github.com/exercism/rust/blob/727f6c5d859965ffecd78d98d61c5a766e3bf8f2/exercises/practice/knapsack/.meta/example.rs) needs to be updated as well.

File: exercises/practice/knapsack/tests/knapsack.rs [hunk 1]
```diff
@@ -31,7 +31,7 @@ fn test_no_items() {
     let max_weight = 100;
     let items = vec![];
```
Comment (dem4ron) [inline exercises/practice/knapsack/tests/knapsack.rs:32]:
```suggestion
    let items = [];
```
I think these `vec!` macros can go too.

**Comment by dem4ron** [comment]
@victor-prokhorov Yes, commits must be squashed, but I think it is enough to do it with the Squash and Merge button.

Also, could you please remove all the other `vec!` macros from the test file?
Thank you

**Comment by ErikSchierboom** [comment]
> but I think it is enough to do it with the Squash and Merge button.

That is correct.

**Review by dem4ron — APPROVED** [review]
Great, thank you!

**Comment by dem4ron** [comment]
@victor-prokhorov one last thing: Could you add yourself as a contributor?

in `knapsack/.meta/config.json` create an array called "contributors" and add your github handle :)

**Review by ErikSchierboom — APPROVED** [review]

**Review by dem4ron — APPROVED** [review]

**Pull request merged**